public/index.html

```diff
@@ -25,7 +25,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>HoneyDo</title>
```

Comment:
Honestly? I'm wondering if "Honey Do" looks better when I see it on the browser tab? What do you think?
![honey-do-favicon-screenshot](https://user-images.githubusercontent.com/47455758/120087658-661e3e80-c0af-11eb-98a4-e79663ec373d.jpg)

Reply:
That makes sense to me.  I'll update that.

Reply:
I can see both working well. I lean `HoneyDo` since the fruit is all one word. 

Whichever one we do, it might be good to consider applying that consistency throughout the app probably.

Reply:
I think the logo is written as "Honey Do" for what's it worth!

Reply:
Ah good point!

Reply:
I see you updated the `<title>` element @zeeatwork 

So I declare you PR APPROVED!!!!